<launch>

    <!-- Depth Lookup Config -->
    <arg name="depthlookup_depth_info_topic" default="/pepper_robot/camera/depth/camera_info"/>
    <arg name="depthlookup_rgb_info_topic" default="/pepper_robot/camera/front/camera_info"/>
    <arg name="depthlookup_image_topic" default="/pepper_robot/sink/depth/image_raw"/>
    <arg name="depthlookup_image_rgb_topic" default="/pepper_robot/sink/front/image_raw"/>
    <arg name="depthlookup_shift_center_y" default="1.25"/>

    <node pkg="clf_perception_vision" type="clf_perception_depth_lookup_objects"
          name="clf_perception_depth_lookup_objects" output="screen">
        <param name="depthlookup_depth_info_topic" type="str" value="$(arg depthlookup_depth_info_topic)"/>
        <param name="depthlookup_rgb_info_topic" type="str" value="$(arg depthlookup_rgb_info_topic)"/>
        <param name="depthlookup_image_topic" type="str" value="$(arg depthlookup_image_topic)"/>
        <param name="depthlookup_image_rgb_topic" type="str" value="$(arg depthlookup_image_rgb_topic)"/>
        <param name="depthlookup_shift_center_y" type="double" value="$(arg depthlookup_shift_center_y)"/>
    </node>
</launch>
